embedded-wallets/connect-blockchain/evm/monad/javascript.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Integrate Embedded Wallets with the Monad Blockchain in JavaScript
+sidebar_label: JavaScript
+image: 'img/embedded-wallets/banners/ethereum.png'
+
+keywords: [monad, web3auth, authentication, blockchain]
+description: 'Integrate Embedded Wallets with the Monad Blockchain in Javascript | Embedded Wallets'
+---
+
+import ChainDetailsMonad from '../../_general-connect-blockchain/_monad.mdx'
+import JsBlockchainMethods from '../../../sdk/js/_ethereum-integration-snippets.mdx'
```

Comment:
The tabbed box displays weird in https://metamask-docs-git-monad-ew-consensys-ddffed67.vercel.app/embedded-wallets/connect-blockchain/evm/monad/javascript/#installation